platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/oauth/OAuthSecurityImpl.java

```diff
@@ -326,7 +329,13 @@ private String getNewAccessToken(
     }
 
     if (response.getStatus() != HttpStatus.SC_OK) {
-      LOGGER.debug("Unable to retrieve system access token. {}", body);
+      sanitizeFormParameters(formParam);
```

Comment:
✏️ There should probably still be a `DEBUG` message about the failure to receive the token and this trace message and the `sanitizeFormParameters` call should all be wrapped in a conditional check if `LOGGER.isTraceEnabled()`. Otherwise, we're going to sanitize every time this occurs but almost never log it.